### ISSUE TITLE
ColorVariable triggers for variable declarations w/ RGBA colors

### ISSUE THREAD

**Issue by peacefulrespite** [brigade/scss-lint#609]
When upgrading to 0.42.0 w/ `Improve ColorVariable to flag uses of rgb/rgba/hsl/hsla` in the release, the `ColorVariable` lint gets triggered for the variable declarations that use `rgba`.

Examples of the variable declarations triggering the failure:

``` scss
$some-variable: rgba(235, 141, 69, 0.8);
$some-other-variable: rgba(234, 130, 51, 1);
$transparent-white-background: rgba(255, 255, 255, 0.6);
$transparent-black: rgba(0, 0, 0, 0.2);
```

Is there something wrong w/ those variable declarations or is this an issue with the linter? Thank you!

Possibly related to #600.

**Comment by sds** [comment]
Hey @peacefulrespite, thanks for your question.

These warnings are expected behavior that was introduced in #569. See #488 for the full discussion.

**Comment by peacefulrespite** [comment]
@sds thanks for the info and links to the discussion and PR. I appreciate it!

**Comment by sds** [comment]
Just circling back to say I misread this issue. This is certainly a bug, as you should be able to use color functions in color variable declarations.

Fixed in #612.
